.env

```diff
@@ -1,23 +1 @@
 NODE_ENV='production'
-BASE_URL=''
```

Comment:
These settings in .env (for production) are unneeded. There are no meaningful overrides here, and all these values get replaced by edx-internal during tubular scripts anyways.